tests/php/AssetControlExtensionTest.php

```diff
@@ -19,9 +20,28 @@ class AssetControlExtensionTest extends SapphireTest
 
     protected static $extra_dataobjects = array(
         VersionedObject::class,
-        TestObject::class
+        TestObject::class,
     );
 
+    protected static $required_extensions = [
+        VersionedObject::class => [
+            Versioned::class,
+        ],
+    ];
+
+    public static function setUpBeforeClass()
+    {
+        try {
+            parent::setUpBeforeClass();
+        } catch (\InvalidArgumentException $e) {
+            static::markTestSkipped('Unable to build databas');
```

Comment:
You've removed versioned from `.travis.yml` setup so when will this class ever be tested?